projects/core/components/notification/notification.directive.ts

```diff
@@ -0,0 +1,80 @@
+import type {OnChanges, OnInit} from '@angular/core';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Directive,
+    HostBinding,
+    inject,
+    Input,
+    ViewEncapsulation,
+} from '@angular/core';
+import type {TuiStringHandler} from '@taiga-ui/cdk/types';
+import {tuiIsString, tuiWithStyles} from '@taiga-ui/cdk/utils/miscellaneous';
+import {tuiButtonOptionsProvider} from '@taiga-ui/core/components/button';
+import {tuiLinkOptionsProvider} from '@taiga-ui/core/components/link';
+import {
+    tuiAppearanceOptionsProvider,
+    TuiWithAppearance,
+} from '@taiga-ui/core/directives/appearance';
+import {TuiIcons, TuiWithIcons} from '@taiga-ui/core/directives/icons';
+
+import {TUI_NOTIFICATION_OPTIONS} from './notification.options';
+
+@Component({
+    standalone: true,
+    template: '',
+    styles: ['@import "@taiga-ui/core/styles/components/notification.less";'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        class: 'tui-notification',
+    },
+})
+class TuiNotificationStyles {}
+
+@Directive({
+    standalone: true,
+    selector: 'tui-notification,a[tuiNotification],button[tuiNotification]',
```

Comment:
Should we remove `tui-notification`  and simplify selector to the following way ?

```suggestion
    selector: '[tuiNotification]',
```

For example
```html
<div tuiNotification />
```